packages/js/components/src/ellipsis-menu/menu-title.tsx

```diff
@@ -1,26 +1,21 @@
 /**
  * External dependencies
  */
-import PropTypes from 'prop-types';
 import { createElement } from '@wordpress/element';
+import React from 'react';
 
 /**
  * `MenuTitle` is another valid Menu child, but this does not have any accessibility attributes associated
  * (so this should not be used in place of the `EllipsisMenu` prop `label`).
- *
- * @param {Object} props
- * @param {Node}   props.children
- * @return {Object} -
  */
-const MenuTitle = ( { children } ) => {
-	return <div className="woocommerce-ellipsis-menu__title">{ children }</div>;
-};
 
-MenuTitle.propTypes = {
+const MenuTitle: React.FC< {
```

Comment:
@moon0326 As Sam commented, we can remove `React.FC` here and use `{ children }: { children: React.ReactNode }` instead. 🙂

Reply:
Hey @moon0326 Would you like to change this?